notebooks/238-deeployd-if/deep-floyd-if.ipynb

```diff
@@ -0,0 +1,783 @@
+{
```

Comment:
<p>I think, given amount of data that is being downloaded and how much time it takes - we need to do some disclaimer like "This notebook requires downloading xxGB of models. It may take time, consider this before starting"</p><p> </p><p>Also, model description is not full. I think main advantage of this model is that it generates picture with realistic text. We never mention that.</p><p> </p><p>Question on a picture: do I undertand correctly that on every stage I have a choice from multiple models and need only one of them? Perhaps we can clarify.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1039/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Added a NOTE section.</p><p>Mentioned the text on image generation feature.</p><p>Descibed model options</p>